charts/kube-prometheus-stack/Chart.yaml

```
@@ -23,7 +23,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 37.2.0
+version: 37.2.1
```

new feature should raise minor version
```suggestion
version: 37.3.0
```